src/v2/providers/pubsub.ts

```diff
@@ -86,29 +103,51 @@ export class Message<T> {
   }
 }
 
-/** The interface published in a Pub/Sub publish subscription. */
+/**
+ * The interface published in a Pub/Sub publish subscription.
+ * @param T - Type representing `Message.data`'s json format
```

Comment:
Suggest "JSON" or `json` if we want the literal.

Reply:
JSON~! 🎉